### DISCUSSION TITLE
Is there an easy way to delete a tenant?

### DISCUSSION THREAD

**Discussion by Erwinvandervalk** [JasperFx/marten#2276] [Q&A]
Hi, Is there a supported way to delete tenants?

I'm now using conjoined multi-tenancy. I see that there are options to clear out the complete database, but is it also possible to delete a single tenant?

Thanks!

Btw, awesome work on marten. Really impressed with how it works and how easy it is to get going!

**Comment by jeremydmiller** [comment]
No, but that's not a crazy pull request to take on. Also possible from pure SQL in the meantime.
